features/admin.extensions.v1/configs/endpoints.ts

```diff
@@ -45,6 +45,7 @@ export const getExtendedFeatureResourceEndpoints = (serverHost: string,
         organizationEndpoint: `${ serverHost }/api/asgardeo-enterprise-login/v1/business-user-login/{organization}`,
         organizationPatchEndpoint: `${ serverHost }/api/asgardeo-enterprise-login/v1/business-user-login`,
         resendEndpoint: `${ serverHost }/api/asgardeo-guest/v1/users/invite/{}/resend`,
+        smsManagement: `${ serverHost }/api/server/v1/notification/sms`,
```

Comment:
Let's not add new endpoints to this, as extensions module is soon to be deprecated.

Reply:
Use the endpoints file inside the component itself.